datasync/chaindatafetcher/kafka/config.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package kafka
+
+type KafkaConfig struct {
+	brokers    []string // brokers is a list of broker URLs.
+	partitions int32    // partitions is the number of partitions of a topic.
+	replicas   int16    // replicas is a replication factor of kafka settings. This is the number of the replicated partitions in the kafka cluster.
+}
+
+var DefaultKafkaConfig = &KafkaConfig{
+	partitions: int32(1),
+	replicas:   int16(1),
+}
```

Comment:
```suggestion
func GetDefaultKafkaConfig() *KafkaConfig {
	return &KafkaConfig{
		partitions: 1,
		replicas:   1,
	}
}
```
It would be best to set these values as const. However, because it is unable to set struct as const, how about declaring it as Get function?

Reply:
Updated it here. 89e762171cb733dc1bf6d942cb4d033cfcf5db7f